hibernate-redis/src/main/java/org/hibernate/cache/redis/AbstractRedisRegionFactory.java

```diff
@@ -92,13 +94,21 @@ public boolean isMinimalPutsEnabledByDefault() {
         return true;
     }
 
+    protected void createJedisClientAndTimestamper(Settings settings, Properties properties) {
+        if (redis == null) {
```

Comment:
well if it's saying that and it is invoked twice it is breaking it's javadoc contract which means there is a bug somewhere. I would at the very least log an error here.